engine/src/main/java/org/terasology/world/internal/WorldProviderCoreImpl.java

```diff
@@ -275,7 +275,7 @@ private void notifyExtraDataChanged(int index, Vector3ic pos, int newData, int o
     public Block getBlock(int x, int y, int z) {
         CoreChunk chunk = chunkProvider.getChunk(Chunks.toChunkPos(x, y, z, new Vector3i()));
         if (chunk != null) {
-            return chunk.getBlock(Chunks.toRelativeX(x), Chunks.toRelativeX(y), Chunks.toRelativeX(z));
```

Comment:
this... is so obvious :see_no_evil: 

thanks for figuring this out! :green_heart: